features/hdr-header-footer.feature

```diff
@@ -58,6 +58,20 @@ Feature: Header and footer behaviors
       | with no    |
 
 
+  @wip
+  Scenario Outline: _Header.end_bookmark()
+    Given a _Header object <with-or-no> header definition as header
+     When I assign bookmark = header.start_bookmark("Target")
+      And I end bookmark by calling header.end_bookmark(bookmark)
```

Comment:
I would reduce this to: `And I call header.end_bookmark(bookmark)`.

Reply:
Fixed